backend/controllers/game-controllers.js

```diff
@@ -51,7 +51,7 @@ function getRoom(req, res) {
 
 async function generateGame(roomId) {
   const commonSongIds = new Set();
-  let commonArtistIds = new Set();
+  const commonArtistIds = new Set();
```

Comment:
change it back to `let`. I intentionally set it to let because in the later lines, I re assign this variable

```
// doing this because I wanted to limit the amount of artists we request data for, to just 10
commonArtistIds = new Set([...commonArtistIds].slice(0, 10));
```


OR alternatively, we can create a new variable to use for the new list of at most 10 artist ids